_data/chats.yml

```diff
@@ -17,14 +14,8 @@ chats:
 - monero-pools:
   matrix: https://matrix.to/#/%23monero-pools:monero.social
 
-- monero-research-lab:
-  matrix: https://matrix.to/#/%23monero-research-lab:monero.social
-
-- monero-translations:
-  matrix: https://matrix.to/#/%23monero-translations:monero.social
-
-- monero-hardware:
-  matrix: https://matrix.to/#/%23monero-hardware:matrix.org
+- monero-support:
+    matrix: https://matrix.to/#/#monero-support:monero.social
 
-- monero-site:
-  matrix: https://matrix.to/#/%23monero-site:monero.social
+- monero-turkce:
+  matrix: https://matrix.to/#/#monero-turkce:monero.social
```

Comment:
would be better suited to display on the localised versions of the page, iirc the xmr .ru matrix group is years old. i've not looked into how easy this can be done. probably best to drop this from this PR for now

Reply:
Alright, removed from this PR and the issue can be tracked from #2432